setup.py

```diff
@@ -62,6 +62,7 @@
         },
         python_requires='>=3',
         install_requires=requirements,
+        include_package_data=True,
```

Comment:
what does this do?

Reply:
without it the templates aren't distributed

Reply:
in particular they're not included in the bdist_wheel. I've added a comment to that effect.